### ISSUE TITLE
Federalist.gov Tab Order

### ISSUE THREAD

**Issue by ShannonBad** [18F/federalist#1618]
#### Description of feature or bug
Federalist.gov tab order does not start with main site navigation and instead starts with page content. Possible accessibility issue.

<TODO>

#### Definition of done
Federalist.gov tab order utilizes main site navigation on all pages, as well as page content.
<TODO>

#### Level of effort
low

**Comment by wslack** [comment]
@ShannonBad can you clarify this with a screenshot pointing to what's going on? I don't observe this on federalist.18f.gov.

**Comment by ShannonBad** [comment]
@wslack the tab order was broken (skipped the navigation) when I posted this, it's now working again.

**Comment by wslack** [comment]
Oh neat!

**Comment by wslack** [comment]
moved to Done?

**Comment by ShannonBad** [comment]
@wslack yes "done", and how do I move to done again?

**Comment by wslack** [comment]
Sorry

I moved this issue to "Done?" in ZenHub to go over next retro as completed.
